src/bundle/Resources/public/js/scripts/button.state.checkbox.toggle.js

```diff
@@ -0,0 +1,27 @@
+(function (global, doc) {
+    const toggleForms = [...doc.querySelectorAll('.ez-toggle-btn-state-checkbox')];
```

Comment:
Due to latest changes in the browsers specification implementation, `doc.querySelectorAll('.ez-toggle-btn-state-checkbox')` is just enough. No need to convert it to an array to use with `forEach` only.